charts/secretsmanagement-operator/values.yaml

```diff
@@ -8,9 +8,9 @@ sidecarVersion: 0.1.0
 sidecarPrereleaseSuffix:
 sidecarImagePullPolicy: IfNotPresent 
   
-# TODO: add TLS to canvas-vault-hc
-vault_addr: "http://canvas-vault-hc.canvas-vault.svc.cluster.local:8200"
+vault_addr: "https://canvas-vault-hc.canvas-vault.svc.cluster.local:8200"
```

Comment:
not so easy here, because the canvasfault.namespace is not defined in the secretsmanagement-operator template. I will have to take a look how to solve this here.

Reply:
In values.yaml itself, it is not possible to use templates. 
So, vault_addr hos to be outsourced into the templates itself.
This is possible in theory, but it will really get unreadable and hard to understand. 
I think the better solution is to leave a reminder in the values.yaml, that changing the namespace requires to change the internal url.